examples/evaluator/golang/simple/evaluate/evaluator.go

```diff
@@ -79,20 +86,35 @@ func Evaluate(p *harness.EvaluatorParams) ([]*pb.Match, error) {
 	return d.results, nil
 }
 
+type collidingMatch struct {
+	id    string
+	score float64
+}
+
 type decollider struct {
 	results     []*pb.Match
-	ticketsUsed map[string]struct{}
+	ticketsUsed map[string]*collidingMatch
 }
 
 func (d *decollider) maybeAdd(m *matchInp) {
 	for _, t := range m.match.GetTickets() {
-		if _, ok := d.ticketsUsed[t.Id]; ok {
+		if cm, ok := d.ticketsUsed[t.Id]; ok {
+			logger.WithFields(logrus.Fields{
+				"match_id":              m.match.GetMatchId(),
+				"ticket_id":             t.GetId(),
+				"match_score":           m.inp.GetScore(),
+				"colliding_match_id":    cm.id,
+				"colliding_match_score": cm.score,
+			}).Info("Higher quality match with colliding ticket found. Rejecting match.")
```

Comment:
Maybe `logger.Debug`?

Reply:
lets keep a higher tracing level for now specially for things that evaluator drops. I would prefer the demo executions to show this log statement by default